packages-exp/analytics-exp/src/api.ts

```diff
@@ -221,8 +221,8 @@ export function setAnalyticsCollectionEnabled(
  * app instance on this device.
  * @public
  * List of recommended event parameters can be found in
- * {@link https://developers.google.com/gtagjs/reference/event
- * | the gtag.js reference documentation}.
+ * {@link https://developers.google.com/gtagjs/reference/ga4-events
```

Comment:
What is ga4?

Reply:
Google Analytics 4, the new version of Google Analytics with App+Web properties and measurement IDs that start with "GA" instead of "UA" for Universal Analytics. The other link was for the older UA reference. All Firebase Analytics properties are GA4.